packages/classes/docs/PAYLOAD_ARRAY.md

```diff
@@ -0,0 +1,23 @@
+# Payload Array
+
+PayloadArrays provide a set of convenient mechanisms for storing and filtering Payloads. This is useful any time sets of Payloads are being stored in memory or processed at one time.
+
+For example:
+
+- `Appliances` use PayloadArrays to store Payloads and ultimately pass them to the Appliance's `invoke` method.
+
+## Attributes
+
+PayloadArrays have the following attributes:
+
+- `length` (Number): The number of Payloads stored in the PayloadArray.
+
+## Methods
+
+PayloadArrays have the following methods:
+
+- `insert(payload)`: Insert a Payload into the PayloadArray while maintaining position order.
+- `indexOfPosition(position)`: Find the index of the first Payload whose position is greater than or equal to this one.
+- `filterByType(type)`: Filter out the Payloads of the specified type (does not modify the PayloadArray).
+- `filterByPosition(start, end?)`: Filter out the Payloads that exist within a specified duration.
```

Comment:
I just realized that both filter descriptions read a little weird. "Filter _out_" sounds like they're being removed from the array, rather than just being selected. I think the method names are fine (it's common), it's just the preposition (or whatever "out" is serving as) that reads weird.

You may also want to clarify the rest of this specific description based on our chatter about only positions inclusively between start/end being included. Maybe "…Payloads whose starting `position` exists within the specified moments." Using the word `duration` may be overloaded here.

Reply:
I updated the description in code but not here (which also makes me realize, maybe I shouldn't actually try to document things here, and instead we should be generating documentation)

Reply:
We should probably be generating docs but I do think this is fine until we are!

Reply:
Borrowed the language from `Array.filter` -- `Create a new PayloadArray containing only Payloads of the specified type.`

Reply:
It doesn't look like the descriptions actually changed?